jdaviz/configs/imviz/plugins/rotate_canvas/rotate_canvas.vue

```diff
@@ -1,64 +1,84 @@
-<template>
-  <j-tray-plugin        
-    description="Rotate viewer canvas to any orientation (note: this does not affect the underlying data)."
-    :link="'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#canvas-rotation'"
-    :popout_button="popout_button">
-
-    <plugin-viewer-select
-      :items="viewer_items"
-      :selected.sync="viewer_selected"
-      label="Viewer"
-      hint="Select viewer."
-    />
-    
-    <v-row>
-      <span style="line-height: 36px">Presets:</span>
-      <j-tooltip tooltipcontent="reset rotation and flip">
-        <v-btn icon @click="reset">
-          <v-icon>mdi-restore</v-icon>
-        </v-btn>
-      </j-tooltip>
-      <j-tooltip v-if="has_wcs" tooltipcontent="north up, east right">
-        <v-btn icon @click="set_north_up_east_right">
-          <img :src="icon_nuer" width="24" class="invert-if-dark" style="opacity: 0.65"/>
-        </v-btn>
-      </j-tooltip>
-      <j-tooltip v-if="has_wcs" tooltipcontent="north up, east left">
-        <v-btn icon @click="set_north_up_east_left">
-          <img :src="icon_nuel" width="24" class="invert-if-dark" style="opacity: 0.65"/>
-        </v-btn>
-      </j-tooltip>
-    </v-row>
-    
-    <v-row>
-      <v-slider
-        v-model="angle"
-        class="align-center"
-        max="180"
-        min="-180"
-        step="1"
-        color="#00617E"
-        track-color="#00617E"
-        thumb-color="#153A4B"
-        hide-details
-      >
-      </v-slider>
-    </v-row>
-
-    <v-row>
-      <v-col>
-        <v-text-field
-          v-model.number="angle"
-          type="number"
-          label="Angle"
-          hint="Rotation angle in degrees clockwise"
-        ></v-text-field>
-      </v-col>
-    </v-row>
-
-    <v-row>
-      <v-switch v-model="flip_horizontal" label="Flip horizontally after rotation"></v-switch>
-    </v-row>
-
-  </j-tray-plugin>
-</template>
+<template>
+  <j-tray-plugin  
+    description="Rotate viewer canvas to any orientation (note: this does not affect the underlying data)."
+    :link="'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#canvas-rotation'"
+    :disabled_msg="isChromium() ? '' : 'Image rotation is not supported by your browser. Please see our docs for more information.'"
```

Comment:
Is there any way to make the `learn more` link still visible when the disabled message is shown, or is that a whole different can of worms? It's disappointing that the link is there when the plugin is active, but not when the plugin is literally telling you to go read the docs.

Reply:
This actually shouldn't be too difficult by modifying `tray_plugin.vue` (the `v-if`/`v-else` probably just needs to moved down to to only have `<slot></slot>` in the `v-else`)

Reply:
Done! 
![image](https://github.com/spacetelescope/jdaviz/assets/25206008/057fd7b3-79d4-48ce-aef8-a2f637138716)